src/models/reposyncstate.ts

```diff
@@ -40,7 +40,7 @@ export class RepoSyncState extends Model {
 	updatedAt: Date;
 	createdAt: Date;
 
-
+	// TODO: why it is only for pullStatus, branchStatus and commitStatus ?!
 	get status(): TaskStatus {
```

Comment:
I tried hard, with my vim, couldn't find the usage. Can you have a look? And :nuke it if not used? 
ps: make sure check handlebar/html as well.

Reply:
Sure, I can do this in my next pr :ifyouknowwhatimean: